hearpreprocess/pipeline.py

```diff
@@ -749,6 +817,7 @@ def requires(self):
         }
 
     def run(self):
+        split_label_count = {key: 0 for key in SPLITS}
         for split_path in self.requires()["data"].workdir.iterdir():
```

Comment:
Assert that split_paths are unique for all data requires, otherwise there could be problems

Reply:
removed iterdir and made it explicit.